deltachat-desktop: build fails

Package deltachat-desktop has a problem. Flagging as broken
Maintainer package -> @dotlambda

![Screenshot from 2022-10-25 10-03-37](https://user-images.githubusercontent.com/48295298/197821064-8f7ef449-9d97-40a6-b848-4dbd16755020.png)

Not an user to this `deltachat-desktop`, just trying to help.

```bash
nix build github:NixOS/nixpkgs/03a00f66fc4e893dccba1579df6d0c83852e1c2c#deltachat-desktop
```

It outputs:
```bash
error: builder for '/nix/store/2l06qpi7428bjyvgdv110vrz7z8276z6-libdeltachat-1.86.0.drv' failed with exit code 2;
       last 10 log lines:
       >      |
       > 3866 |     Box::from_raw(lot);
       >      |     ^^^^^^^^^^^^^^^^^^^
       >      |
       >      = note: call `drop(from_raw(ptr))` if you intend to drop the `Box`
       >
       > error: could not compile `deltachat_ffi` due to 9 previous errors
       > make[2]: *** [CMakeFiles/lib_deltachat.dir/build.make:74: target/release/libdeltachat.so] Error 101
       > make[1]: *** [CMakeFiles/Makefile2:83: CMakeFiles/lib_deltachat.dir/all] Error 2
       > make: *** [Makefile:136: all] Error 2
       For full logs, run 'nix log /nix/store/2l06qpi7428bjyvgdv110vrz7z8276z6-libdeltachat-1.86.0.drv'.
error: 1 dependencies of derivation '/nix/store/h5s47s864jyhs9i9qcsg0zmgwwc3d64j-deltachat-desktop-1.30.1.drv' failed to build
```

<details>
<summary>Click to expand to see the full nix log /nix/store/2l06qpi7428bjyvgdv110vrz7z8276z6-libdeltachat-1.86.0.drv </summary>
@nix { "action": "setPhase", "phase": "unpackPhase" }
unpacking sources
unpacking source archive /nix/store/7vlpkq45p8h8clw0x3jk2wja1pj0i5am-source
source root is source
Executing cargoSetupPostUnpackHook
unpacking source archive /nix/store/7aqr3y37kyssr9s257dgs0gyrfm62xzw-libdeltachat-1.86.0-vendor.tar.gz
Finished cargoSetupPostUnpackHook
unpackPhase completed in 1 minutes 39 seconds
@nix { "action": "setPhase", "phase": "patchPhase" }
patching sources
applying patch /nix/store/8h1gakb1x8iq8bknwwjqa1sv92ghjgwq-no-static-lib.patch
patching file CMakeLists.txt
patching file deltachat-ffi/Cargo.toml
Executing cargoSetupPostPatchHook
Validating consistency between /build/source/Cargo.lock and /build/libdeltachat-1.86.0-vendor.tar.gz/Cargo.lock
Finished cargoSetupPostPatchHook
@nix { "action": "setPhase", "phase": "configurePhase" }
configuring
fixing cmake files...
cmake flags: -DCMAKE_FIND_USE_SYSTEM_PACKAGE_REGISTRY=OFF -DCMAKE_FIND_USE_PACKAGE_REGISTRY=OFF -DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON -DCMAKE_BUILD_TYPE=Release -DBUILD_TESTING=OFF -DCMAKE_INSTALL_LOCALEDIR=/nix/store/1f3vwj869vqxhncxv490k7rc6ixj5fj8-libdeltachat-1.86.0/share/locale -DCMAKE_INSTALL_LIBEXECDIR=/nix/store/1f3vwj869vqxhncxv490k7rc6ixj5fj8-libdeltachat-1.86.0/libexec -DCMAKE_INSTALL_LIBDIR=/nix/store/1f3vwj869vqxhncxv490k7rc6ixj5fj8-libdeltachat-1.86.0/lib -DCMAKE_INSTALL_DOCDIR=/nix/store/1f3vwj869vqxhncxv490k7rc6ixj5fj8-libdeltachat-1.86.0/share/doc/deltachat -DCMAKE_INSTALL_INFODIR=/nix/store/1f3vwj869vqxhncxv490k7rc6ixj5fj8-libdeltachat-1.86.0/share/info -DCMAKE_INSTALL_MANDIR=/nix/store/1f3vwj869vqxhncxv490k7rc6ixj5fj8-libdeltachat-1.86.0/share/man -DCMAKE_INSTALL_OLDINCLUDEDIR=/nix/store/1f3vwj869vqxhncxv490k7rc6ixj5fj8-libdeltachat-1.86.0/include -DCMAKE_INSTALL_INCLUDEDIR=/nix/store/1f3vwj869vqxhncxv490k7rc6ixj5fj8-libdeltachat-1.86.0/include -DCMAKE_INSTALL_SBINDIR=/nix/store/1f3vwj869vqxhncxv490k7rc6ixj5fj8-libdeltachat-1.86.0/sbin -DCMAKE_INSTALL_BINDIR=/nix/store/1f3vwj869vqxhncxv490k7rc6ixj5fj8-libdeltachat-1.86.0/bin -DCMAKE_INSTALL_NAME_DIR=/nix/store/1f3vwj869vqxhncxv490k7rc6ixj5fj8-libdeltachat-1.86.0/lib -DCMAKE_POLICY_DEFAULT_CMP0025=NEW -DCMAKE_OSX_SYSROOT= -DCMAKE_FIND_FRAMEWORK=LAST -DCMAKE_STRIP=/nix/store/0vln6gxc9b2wr9yhbn0aspy6badlj638-gcc-wrapper-11.3.0/bin/strip -DCMAKE_RANLIB=/nix/store/0vln6gxc9b2wr9yhbn0aspy6badlj638-gcc-wrapper-11.3.0/bin/ranlib -DCMAKE_AR=/nix/store/0vln6gxc9b2wr9yhbn0aspy6badlj638-gcc-wrapper-11.3.0/bin/ar -DCMAKE_C_COMPILER=gcc -DCMAKE_CXX_COMPILER=g++ -DCMAKE_INSTALL_PREFIX=/nix/store/1f3vwj869vqxhncxv490k7rc6ixj5fj8-libdeltachat-1.86.0  
-- The C compiler identification is GNU 11.3.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /nix/store/0vln6gxc9b2wr9yhbn0aspy6badlj638-gcc-wrapper-11.3.0/bin/gcc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Configuring done
-- Generating done
[33mCMake Warning:
  Manually-specified variables were not used by the project:

    BUILD_TESTING
    CMAKE_CXX_COMPILER
    CMAKE_EXPORT_NO_PACKAGE_REGISTRY
    CMAKE_FIND_USE_PACKAGE_REGISTRY
    CMAKE_FIND_USE_SYSTEM_PACKAGE_REGISTRY
    CMAKE_INSTALL_NAME_DIR
    CMAKE_POLICY_DEFAULT_CMP0025

[0m
-- Build files have been written to: /build/source/build
cmake: enabled parallel building
@nix { "action": "setPhase", "phase": "buildPhase" }
building
build flags: -j4 SHELL=/nix/store/dd3713mm8wql4r2d5jxx0f58g16nfm4h-bash-5.1-p16/bin/bash
[100%] [34m[1mGenerating target/release/libdeltachat.so, target/release/pkgconfig/deltachat.pc[0m
[0m[0m[1m[32m   Compiling[0m version_check v0.9.4
[0m[0m[1m[32m   Compiling[0m proc-macro2 v1.0.39
[0m[0m[1m[32m   Compiling[0m cfg-if v1.0.0
[0m[0m[1m[32m   Compiling[0m unicode-ident v1.0.0
[0m[0m[1m[32m   Compiling[0m syn v1.0.95
[0m[0m[1m[32m   Compiling[0m libc v0.2.126
[0m[0m[1m[32m   Compiling[0m autocfg v1.1.0
[0m[0m[1m[32m   Compiling[0m memchr v2.5.0
[0m[0m[1m[32m   Compiling[0m typenum v1.15.0
[0m[0m[1m[32m   Compiling[0m futures-core v0.3.21
[0m[0m[1m[32m   Compiling[0m spin v0.5.2
[0m[0m[1m[32m   Compiling[0m serde_derive v1.0.137
[0m[0m[1m[32m   Compiling[0m log v0.4.17
[0m[0m[1m[32m   Compiling[0m once_cell v1.12.0
[0m[0m[1m[32m   Compiling[0m serde v1.0.137
[0m[0m[1m[32m   Compiling[0m pin-project-lite v0.2.9
[0m[0m[1m[32m   Compiling[0m futures-io v0.3.21
[0m[0m[1m[32m   Compiling[0m opaque-debug v0.3.0
[0m[0m[1m[32m   Compiling[0m cc v1.0.73
[0m[0m[1m[32m   Compiling[0m event-listener v2.5.2
[0m[0m[1m[32m   Compiling[0m slab v0.4.6
[0m[0m[1m[32m   Compiling[0m cache-padded v1.2.0
[0m[0m[1m[32m   Compiling[0m ppv-lite86 v0.2.16
[0m[0m[1m[32m   Compiling[0m parking v2.0.0
[0m[0m[1m[32m   Compiling[0m fastrand v1.7.0
[0m[0m[1m[32m   Compiling[0m subtle v2.4.1
[0m[0m[1m[32m   Compiling[0m waker-fn v1.1.0
[0m[0m[1m[32m   Compiling[0m pkg-config v0.3.25
[0m[0m[1m[32m   Compiling[0m bitflags v1.3.2
[0m[0m[1m[32m   Compiling[0m getrandom v0.1.16
[0m[0m[1m[32m   Compiling[0m pin-utils v0.1.0
[0m[0m[1m[32m   Compiling[0m crossbeam-utils v0.8.8
[0m[0m[1m[32m   Compiling[0m block-padding v0.2.1
[0m[0m[1m[32m   Compiling[0m base64 v0.13.0
[0m[0m[1m[32m   Compiling[0m percent-encoding v2.1.0
[0m[0m[1m[32m   Compiling[0m matches v0.1.9
[0m[0m[1m[32m   Compiling[0m smallvec v1.8.0
[0m[0m[1m[32m   Compiling[0m signal-hook v0.3.14
[0m[0m[1m[32m   Compiling[0m ryu v1.0.10
[0m[0m[1m[32m   Compiling[0m tinyvec_macros v0.1.0
[0m[0m[1m[32m   Compiling[0m async-task v4.2.0
[0m[0m[1m[32m   Compiling[0m futures-channel v0.3.21
[0m[0m[1m[32m   Compiling[0m futures-task v0.3.21
[0m[0m[1m[32m   Compiling[0m atomic-waker v1.0.0
[0m[0m[1m[32m   Compiling[0m unicode-bidi v0.3.8
[0m[0m[1m[32m   Compiling[0m byteorder v1.4.3
[0m[0m[1m[32m   Compiling[0m futures-util v0.3.21
[0m[0m[1m[32m   Compiling[0m cpufeatures v0.2.2
[0m[0m[1m[32m   Compiling[0m futures-sink v0.3.21
[0m[0m[1m[32m   Compiling[0m lexical-core v0.7.6
[0m[0m[1m[32m   Compiling[0m proc-macro-hack v0.5.19
[0m[0m[1m[32m   Compiling[0m static_assertions v1.1.0
[0m[0m[1m[32m   Compiling[0m arrayvec v0.5.2
[0m[0m[1m[32m   Compiling[0m regex-syntax v0.6.26
[0m[0m[1m[32m   Compiling[0m unicode-xid v0.2.3
[0m[0m[1m[32m   Compiling[0m async-trait v0.1.53
[0m[0m[1m[32m   Compiling[0m const_fn v0.4.9
[0m[0m[1m[32m   Compiling[0m anyhow v1.0.57
[0m[0m[1m[32m   Compiling[0m cpuid-bool v0.2.0
[0m[0m[1m[32m   Compiling[0m openssl v0.10.40
[0m[0m[1m[32m   Compiling[0m parking_lot_core v0.9.3
[0m[0m[1m[32m   Compiling[0m scopeguard v1.1.0
[0m[0m[1m[32m   Compiling[0m encoding_index_tests v0.1.4
[0m[0m[1m[32m   Compiling[0m adler v1.0.2
[0m[0m[1m[32m   Compiling[0m foreign-types-shared v0.1.1
[0m[0m[1m[32m   Compiling[0m encoding_rs v0.8.31
[0m[0m[1m[32m   Compiling[0m crossbeam-queue v0.3.5
[0m[0m[1m[32m   Compiling[0m native-tls v0.2.10
[0m[0m[1m[32m   Compiling[0m crc32fast v1.3.2
[0m[0m[1m[32m   Compiling[0m serde_json v1.0.81
[0m[0m[1m[32m   Compiling[0m fnv v1.0.7
[0m[0m[1m[32m   Compiling[0m itoa v1.0.2
[0m[0m[1m[32m   Compiling[0m strsim v0.9.3
[0m[0m[1m[32m   Compiling[0m openssl-probe v0.1.5
[0m[0m[1m[32m   Compiling[0m ident_case v1.0.1
[0m[0m[1m[32m   Compiling[0m radium v0.5.3
[0m[0m[1m[32m   Compiling[0m http-types v2.12.0
[0m[0m[1m[32m   Compiling[0m match_cfg v0.1.0
[0m[0m[1m[32m   Compiling[0m libm v0.2.2
[0m[0m[1m[32m   Compiling[0m parking_lot_core v0.8.5
[0m[0m[1m[32m   Compiling[0m heck v0.4.0
[0m[0m[1m[32m   Compiling[0m httparse v1.7.1
[0m[0m[1m[32m   Compiling[0m vcpkg v0.2.15
[0m[0m[1m[32m   Compiling[0m infer v0.2.3
[0m[0m[1m[32m   Compiling[0m hex v0.4.3
[0m[0m[1m[32m   Compiling[0m wyz v0.2.0
[0m[0m[1m[32m   Compiling[0m tap v1.0.1
[0m[0m[1m[32m   Compiling[0m base64 v0.12.3
[0m[0m[1m[32m   Compiling[0m funty v1.1.0
[0m[0m[1m[32m   Compiling[0m version_check v0.1.5
[0m[0m[1m[32m   Compiling[0m data-encoding v2.3.2
[0m[0m[1m[32m   Compiling[0m stable_deref_trait v1.2.0
[0m[0m[1m[32m   Compiling[0m quick-error v1.2.3
[0m[0m[1m[32m   Compiling[0m linked-hash-map v0.5.4
[0m[0m[1m[32m   Compiling[0m ipnet v2.5.0
[0m[0m[1m[32m   Compiling[0m signature v1.5.0
[0m[0m[1m[32m   Compiling[0m ascii_utils v0.9.3
[0m[0m[1m[32m   Compiling[0m mime v0.3.16
[0m[0m[1m[32m   Compiling[0m derive_builder v0.9.0
[0m[0m[1m[32m   Compiling[0m adler32 v1.2.0
[0m[0m[1m[32m   Compiling[0m Inflector v0.11.4
[0m[0m[1m[32m   Compiling[0m rustversion v1.0.6
[0m[0m[1m[32m   Compiling[0m gimli v0.26.1
[0m[0m[1m[32m   Compiling[0m crc24 v0.1.6
[0m[0m[1m[32m   Compiling[0m safemem v0.3.3
[0m[0m[1m[32m   Compiling[0m base64 v0.11.0
[0m[0m[1m[32m   Compiling[0m color_quant v1.1.0
[0m[0m[1m[32m   Compiling[0m rustc-demangle v0.1.21
[0m[0m[1m[32m   Compiling[0m keccak v0.1.2
[0m[0m[1m[32m   Compiling[0m cfg-if v0.1.10
[0m[0m[1m[32m   Compiling[0m fallible-iterator v0.2.0
[0m[0m[1m[32m   Compiling[0m weezl v0.1.6
[0m[0m[1m[32m   Compiling[0m fallible-streaming-iterator v0.1.9
[0m[0m[1m[32m   Compiling[0m jpeg-decoder v0.2.6
[0m[0m[1m[32m   Compiling[0m circular v0.3.0
[0m[0m[1m[32m   Compiling[0m quoted_printable v0.4.5
[0m[0m[1m[32m   Compiling[0m mutate_once v0.1.1
[0m[0m[1m[32m   Compiling[0m unicode-width v0.1.9
[0m[0m[1m[32m   Compiling[0m bufstream v0.1.4
[0m[0m[1m[32m   Compiling[0m smawk v0.3.1
[0m[0m[1m[32m   Compiling[0m bytemuck v1.9.1
[0m[0m[1m[32m   Compiling[0m bitfield v0.13.2
[0m[0m[1m[32m   Compiling[0m entities v1.0.1
[0m[0m[1m[32m   Compiling[0m deltachat_ffi v1.86.0 (/build/source/deltachat-ffi)
[0m[0m[1m[32m   Compiling[0m qrcodegen v1.8.0
[0m[0m[1m[32m   Compiling[0m tagger v4.3.3
[0m[0m[1m[32m   Compiling[0m rust-hsluv v0.1.4
[0m[0m[1m[32m   Compiling[0m strum v0.24.0
[0m[0m[1m[32m   Compiling[0m humansize v1.1.1
[0m[0m[1m[32m   Compiling[0m termcolor v1.1.3
[0m[0m[1m[32m   Compiling[0m instant v0.1.12
[0m[0m[1m[32m   Compiling[0m generic-array v0.14.5
[0m[0m[1m[32m   Compiling[0m value-bag v1.0.0-alpha.9
[0m[0m[1m[32m   Compiling[0m standback v0.2.17
[0m[0m[1m[32m   Compiling[0m unicase v2.6.0
[0m[0m[1m[32m   Compiling[0m nom v5.1.2
[0m[0m[1m[32m   Compiling[0m time v0.2.27
[0m[0m[1m[32m   Compiling[0m ahash v0.7.6
[0m[0m[1m[32m   Compiling[0m cookie v0.14.4
[0m[0m[1m[32m   Compiling[0m proc-macro-error-attr v1.0.4
[0m[0m[1m[32m   Compiling[0m proc-macro-error v1.0.4
[0m[0m[1m[32m   Compiling[0m nom v6.1.2
[0m[0m[1m[32m   Compiling[0m email v0.0.21 (https://github.com/deltachat/rust-email?branch=master#25702df9)
[0m[0m[1m[32m   Compiling[0m autocfg v0.1.8
[0m[0m[1m[32m   Compiling[0m num-traits v0.2.15
[0m[0m[1m[32m   Compiling[0m num-integer v0.1.45
[0m[0m[1m[32m   Compiling[0m lock_api v0.4.7
[0m[0m[1m[32m   Compiling[0m num-iter v0.1.43
[0m[0m[1m[32m   Compiling[0m num-bigint v0.2.6
[0m[0m[1m[32m   Compiling[0m num-rational v0.4.0
[0m[0m[1m[32m   Compiling[0m lazy_static v1.4.0
[0m[0m[1m[32m   Compiling[0m tokio v1.18.2
[0m[0m[1m[32m   Compiling[0m async-mutex v1.4.0
[0m[0m[1m[32m   Compiling[0m async-lock v2.5.0
[0m[0m[1m[32m   Compiling[0m simple-mutex v1.1.5
[0m[0m[1m[32m   Compiling[0m concurrent-queue v1.2.2
[0m[0m[1m[32m   Compiling[0m form_urlencoded v1.0.1
[0m[0m[1m[32m   Compiling[0m openssl-sys v0.9.73
[0m[0m[1m[32m   Compiling[0m backtrace v0.3.65
[0m[0m[1m[32m   Compiling[0m clear_on_drop v0.2.5
[0m[0m[1m[32m   Compiling[0m tinyvec v1.6.0
[0m[0m[1m[32m   Compiling[0m encoding-index-singlebyte v1.20141219.5
[0m[0m[1m[32m   Compiling[0m encoding-index-korean v1.20141219.5
[0m[0m[1m[32m   Compiling[0m encoding-index-japanese v1.20141219.5
[0m[0m[1m[32m   Compiling[0m encoding-index-simpchinese v1.20141219.5
[0m[0m[1m[32m   Compiling[0m encoding-index-tradchinese v1.20141219.5
[0m[0m[1m[32m   Compiling[0m miniz_oxide v0.5.3
[0m[0m[1m[32m   Compiling[0m foreign-types v0.3.2
[0m[0m[1m[32m   Compiling[0m libsqlite3-sys v0.24.2
[0m[0m[1m[32m   Compiling[0m nom v4.2.3
[0m[0m[1m[32m   Compiling[0m lru-cache v0.1.2
[0m[0m[1m[32m   Compiling[0m ed25519 v1.5.2
[0m[0m[1m[32m   Compiling[0m fast_chemail v0.9.6
[0m[0m[1m[32m   Compiling[0m deflate v1.0.0
[0m[0m[1m[32m   Compiling[0m try_from v0.3.2
[0m[0m[1m[32m   Compiling[0m gif v0.11.3
[0m[0m[1m[32m   Compiling[0m addr2line v0.17.0
[0m[0m[1m[32m   Compiling[0m kamadak-exif v0.5.4
[0m[0m[1m[32m   Compiling[0m escaper v0.1.1
[0m[0m[1m[32m   Compiling[0m num-bigint-dig v0.6.1
[0m[0m[1m[32m   Compiling[0m async-dup v1.2.2
[0m[0m[1m[32m   Compiling[0m unicode-normalization v0.1.19
[0m[0m[1m[32m   Compiling[0m encoding v0.2.33
[0m[0m[1m[32m   Compiling[0m async-channel v1.6.1
[0m[0m[1m[32m   Compiling[0m quote v1.0.18
[0m[0m[1m[32m   Compiling[0m futures-lite v1.12.0
[0m[0m[1m[32m   Compiling[0m aho-corasick v0.7.18
[0m[0m[1m[32m   Compiling[0m object v0.28.4
[0m[0m[1m[32m   Compiling[0m buf_redux v0.8.4
[0m[0m[1m[32m   Compiling[0m quick-xml v0.23.0
[0m[0m[1m[32m   Compiling[0m getrandom v0.2.6
[0m[0m[1m[32m   Compiling[0m socket2 v0.4.4
[0m[0m[1m[32m   Compiling[0m num_cpus v1.13.1
[0m[0m[1m[32m   Compiling[0m signal-hook-registry v1.4.0
[0m[0m[1m[32m   Compiling[0m time v0.1.44
[0m[0m[1m[32m   Compiling[0m hostname v0.3.1
[0m[0m[1m[32m   Compiling[0m filetime v0.2.16
[0m[0m[1m[32m   Compiling[0m idna v0.2.3
[0m[0m[1m[32m   Compiling[0m flate2 v1.0.24
[0m[0m[1m[32m   Compiling[0m png v0.17.5
[0m[0m[1m[32m   Compiling[0m bitvec v0.19.6
[0m[0m[1m[32m   Compiling[0m charset v0.1.3
[0m[0m[1m[32m   Compiling[0m mime_guess v2.0.4
[0m[0m[1m[32m   Compiling[0m stop-token v0.7.0
[0m[0m[1m[32m   Compiling[0m regex v1.5.6
[0m[0m[1m[32m   Compiling[0m async-executor v1.4.1
[0m[0m[1m[32m   Compiling[0m blocking v1.2.0
[0m[0m[1m[32m   Compiling[0m rand_core v0.6.3
[0m[0m[1m[32m   Compiling[0m uuid v0.8.2
[0m[0m[1m[32m   Compiling[0m rand_core v0.5.1
[0m[0m[1m[32m   Compiling[0m dashmap v4.0.2
[0m[0m[1m[32m   Compiling[0m resolv-conf v0.7.0
[0m[0m[1m[32m   Compiling[0m parking_lot v0.12.1
[0m[0m[1m[32m   Compiling[0m parking_lot v0.11.2
[0m[0m[1m[32m   Compiling[0m byte-pool v0.2.3
[0m[0m[1m[32m   Compiling[0m digest v0.9.0
[0m[0m[1m[32m   Compiling[0m cipher v0.2.5
[0m[0m[1m[32m   Compiling[0m block-buffer v0.9.0
[0m[0m[1m[32m   Compiling[0m universal-hash v0.4.1
[0m[0m[1m[32m   Compiling[0m crypto-mac v0.10.1
[0m[0m[1m[32m   Compiling[0m aead v0.3.2
[0m[0m[1m[32m   Compiling[0m block-buffer v0.10.2
[0m[0m[1m[32m   Compiling[0m crypto-common v0.1.3
[0m[0m[1m[32m   Compiling[0m block-cipher v0.8.0
[0m[0m[1m[32m   Compiling[0m zip v0.6.2
[0m[0m[1m[32m   Compiling[0m mailparse v0.13.8
[0m[0m[1m[32m   Compiling[0m pem v0.8.3
[0m[0m[1m[32m   Compiling[0m sanitize-filename v0.4.0
[0m[0m[1m[32m   Compiling[0m os_type v2.4.0
[0m[0m[1m[32m   Compiling[0m rand_chacha v0.3.1
[0m[0m[1m[32m   Compiling[0m hashbrown v0.11.2
[0m[0m[1m[32m   Compiling[0m rand_chacha v0.2.2
[0m[0m[1m[32m   Compiling[0m scheduled-thread-pool v0.2.6
[0m[0m[1m[32m   Compiling[0m synstructure v0.12.6
[0m[0m[1m[32m   Compiling[0m darling_core v0.10.2
[0m[0m[1m[32m   Compiling[0m aes-soft v0.6.4
[0m[0m[1m[32m   Compiling[0m ctr v0.6.0
[0m[0m[1m[32m   Compiling[0m cast5 v0.9.0
[0m[0m[1m[32m   Compiling[0m cfb-mode v0.6.0
[0m[0m[1m[32m   Compiling[0m des v0.6.0
[0m[0m[1m[32m   Compiling[0m blowfish v0.7.0
[0m[0m[1m[32m   Compiling[0m twofish v0.5.0
[0m[0m[1m[32m   Compiling[0m sha2 v0.9.9
[0m[0m[1m[32m   Compiling[0m sha3 v0.9.1
[0m[0m[1m[32m   Compiling[0m sha-1 v0.9.8
[0m[0m[1m[32m   Compiling[0m ripemd160 v0.9.1
[0m[0m[1m[32m   Compiling[0m md-5 v0.9.1
[0m[0m[1m[32m   Compiling[0m polyval v0.4.5
[0m[0m[1m[32m   Compiling[0m hmac v0.10.1
[0m[0m[1m[32m   Compiling[0m digest v0.10.3
[0m[0m[1m[32m   Compiling[0m block-modes v0.6.1
[0m[0m[1m[32m   Compiling[0m chrono v0.4.19
[0m[0m[1m[32m   Compiling[0m ctor v0.1.22
[0m[0m[1m[32m   Compiling[0m thiserror-impl v1.0.31
[0m[0m[1m[32m   Compiling[0m futures-macro v0.3.21
[0m[0m[1m[32m   Compiling[0m openssl-macros v0.1.0
[0m[0m[1m[32m   Compiling[0m time-macros-impl v0.1.2
[0m[0m[1m[32m   Compiling[0m pin-project-internal v1.0.10
[0m[0m[1m[32m   Compiling[0m enum-as-inner v0.4.0
[0m[0m[1m[32m   Compiling[0m num-derive v0.3.3
[0m[0m[1m[32m   Compiling[0m strum_macros v0.24.0
[0m[0m[1m[32m   Compiling[0m deltachat_derive v2.0.0 (/build/source/deltachat_derive)
[0m[0m[1m[32m   Compiling[0m unicode-linebreak v0.1.2
[0m[0m[1m[32m   Compiling[0m rand v0.8.5
[0m[0m[1m[32m   Compiling[0m rand v0.7.3
[0m[0m[1m[32m   Compiling[0m hashlink v0.7.0
[0m[0m[1m[32m   Compiling[0m imap-proto v0.14.3
[0m[0m[1m[32m   Compiling[0m ouroboros_macro v0.9.5
[0m[0m[1m[32m   Compiling[0m aes v0.6.0
[0m[0m[1m[32m   Compiling[0m zeroize_derive v1.3.2
[0m[0m[1m[32m   Compiling[0m ghash v0.3.1
[0m[0m[1m[32m   Compiling[0m hkdf v0.10.0
[0m[0m[1m[32m   Compiling[0m sha-1 v0.10.0
[0m[0m[1m[32m   Compiling[0m sha2 v0.10.2
[0m[0m[1m[32m   Compiling[0m image v0.24.2
[0m[0m[1m[32m   Compiling[0m darling_macro v0.10.2
[0m[0m[1m[32m   Compiling[0m simple_asn1 v0.4.1
[0m[0m[1m[32m   Compiling[0m thiserror v1.0.31
[0m[0m[1m[32m   Compiling[0m time-macros v0.1.1
[0m[0m[1m[32m   Compiling[0m pin-project v1.0.10
[0m[0m[1m[32m   Compiling[0m rusqlite v0.27.0
[0m[0m[1m[32m   Compiling[0m aes-gcm v0.8.0
[0m[0m[1m[32m   Compiling[0m zeroize v1.3.0
[0m[0m[1m[32m   Compiling[0m ouroboros v0.9.5
[0m[0m[1m[32m   Compiling[0m darling v0.10.2
[0m[0m[1m[32m   Compiling[0m encoded-words v0.1.0 (https://github.com/async-email/encoded-words?branch=master#e878b692)
[0m[0m[1m[32m   Compiling[0m futures-executor v0.3.21
[0m[0m[1m[32m   Compiling[0m url v2.2.2
[0m[0m[1m[32m   Compiling[0m serde_urlencoded v0.7.1
[0m[0m[1m[32m   Compiling[0m serde_qs v0.8.5
[0m[0m[1m[32m   Compiling[0m config v0.10.1
[0m[0m[1m[32m   Compiling[0m toml v0.5.9
[0m[0m[1m[32m   Compiling[0m uuid v1.1.1
[0m[0m[1m[32m   Compiling[0m curve25519-dalek v3.2.1
[0m[0m[1m[32m   Compiling[0m derive_builder_core v0.9.0
[0m[0m[1m[32m   Compiling[0m polling v2.2.0
[0m[0m[1m[32m   Compiling[0m kv-log-macro v1.0.7
[0m[0m[1m[32m   Compiling[0m lettre v0.9.2 (https://github.com/deltachat/lettre?branch=master#96555ec4)
[0m[0m[1m[32m   Compiling[0m r2d2 v0.8.9
[0m[0m[1m[32m   Compiling[0m futures v0.3.21
[0m[0m[1m[32m   Compiling[0m trust-dns-proto v0.21.2
[0m[0m[1m[32m   Compiling[0m deadpool v0.7.0
[0m[0m[1m[32m   Compiling[0m human-panic v1.0.3
[0m[0m[1m[32m   Compiling[0m x25519-dalek v1.2.0
[0m[0m[1m[32m   Compiling[0m ed25519-dalek v1.0.1
[0m[0m[1m[32m   Compiling[0m rsa v0.3.0
[0m[0m[1m[32m   Compiling[0m textwrap v0.15.0
[0m[0m[1m[32m   Compiling[0m async-io v1.7.0
[0m[0m[1m[32m   Compiling[0m lettre_email v0.9.2 (https://github.com/deltachat/lettre?branch=master#96555ec4)
[0m[0m[1m[32m   Compiling[0m r2d2_sqlite v0.20.0
[0m[0m[1m[32m   Compiling[0m trust-dns-resolver v0.21.2
[0m[0m[1m[32m   Compiling[0m pgp v0.7.2
[0m[0m[1m[32m   Compiling[0m async-global-executor v2.0.4
[0m[0m[1m[32m   Compiling[0m async-process v1.4.0
[0m[0m[1m[32m   Compiling[0m async-std v1.11.0
[0m[0m[1m[32m   Compiling[0m async-native-tls v0.3.3
[0m[0m[1m[32m   Compiling[0m fast-socks5 v0.4.3
[0m[0m[1m[32m   Compiling[0m async-std-resolver v0.21.2
[0m[0m[1m[32m   Compiling[0m async-tar v0.4.2
[0m[0m[1m[32m   Compiling[0m async-imap v0.5.0 (https://github.com/async-email/async-imap#d07ea401)
[0m[0m[1m[32m   Compiling[0m async-smtp v0.4.0 (https://github.com/async-email/async-smtp?branch=master#d6b947cf)
[0m[0m[1m[32m   Compiling[0m async-h1 v2.3.3
[0m[0m[1m[32m   Compiling[0m http-client v6.5.2
[0m[0m[1m[32m   Compiling[0m surf v2.3.2
[0m[0m[1m[32m   Compiling[0m deltachat v1.86.0 (/build/source)
[0m[1m[38;5;9merror[0m[0m[1m: unused return value of `Box::<T>::from_raw` that must be used[0m
[0m   [0m[0m[1m[38;5;12m--> [0m[0mdeltachat-ffi/src/lib.rs:154:5[0m
[0m    [0m[0m[1m[38;5;12m|[0m
[0m[1m[38;5;12m154[0m[0m [0m[0m[1m[38;5;12m|[0m[0m [0m[0m    Box::from_raw(context);[0m
[0m    [0m[0m[1m[38;5;12m| [0m[0m    [0m[0m[1m[38;5;9m^^^^^^^^^^^^^^^^^^^^^^^[0m
[0m    [0m[0m[1m[38;5;12m|[0m
[0m[1m[38;5;10mnote[0m[0m: the lint level is defined here[0m
[0m   [0m[0m[1m[38;5;12m--> [0m[0mdeltachat-ffi/src/lib.rs:1:9[0m
[0m    [0m[0m[1m[38;5;12m|[0m
[0m[1m[38;5;12m1[0m[0m   [0m[0m[1m[38;5;12m|[0m[0m [0m[0m#![deny(unused, clippy::all)][0m
[0m    [0m[0m[1m[38;5;12m| [0m[0m        [0m[0m[1m[38;5;10m^^^^^^[0m
[0m    [0m[0m[1m[38;5;12m= [0m[0m[1mnote[0m[0m: `#[deny(unused_must_use)]` implied by `#[deny(unused)]`[0m
[0m    [0m[0m[1m[38;5;12m= [0m[0m[1mnote[0m[0m: call `drop(from_raw(ptr))` if you intend to drop the `Box`[0m

[0m[1m[38;5;9merror[0m[0m[1m: unused return value of `Box::<T>::from_raw` that must be used[0m
[0m   [0m[0m[1m[38;5;12m--> [0m[0mdeltachat-ffi/src/lib.rs:448:5[0m
[0m    [0m[0m[1m[38;5;12m|[0m
[0m[1m[38;5;12m448[0m[0m [0m[0m[1m[38;5;12m|[0m[0m [0m[0m    Box::from_raw(a);[0m
[0m    [0m[0m[1m[38;5;12m| [0m[0m    [0m[0m[1m[38;5;9m^^^^^^^^^^^^^^^^^[0m
[0m    [0m[0m[1m[38;5;12m|[0m
[0m    [0m[0m[1m[38;5;12m= [0m[0m[1mnote[0m[0m: call `drop(from_raw(ptr))` if you intend to drop the `Box`[0m

[0m[1m[38;5;9merror[0m[0m[1m: unused return value of `Box::<T>::from_raw` that must be used[0m
[0m   [0m[0m[1m[38;5;12m--> [0m[0mdeltachat-ffi/src/lib.rs:668:5[0m
[0m    [0m[0m[1m[38;5;12m|[0m
[0m[1m[38;5;12m668[0m[0m [0m[0m[1m[38;5;12m|[0m[0m [0m[0m    Box::from_raw(emitter);[0m
[0m    [0m[0m[1m[38;5;12m| [0m[0m    [0m[0m[1m[38;5;9m^^^^^^^^^^^^^^^^^^^^^^^[0m
[0m    [0m[0m[1m[38;5;12m|[0m
[0m    [0m[0m[1m[38;5;12m= [0m[0m[1mnote[0m[0m: call `drop(from_raw(ptr))` if you intend to drop the `Box`[0m

[0m[1m[38;5;9merror[0m[0m[1m: unused return value of `Box::<T>::from_raw` that must be used[0m
[0m    [0m[0m[1m[38;5;12m--> [0m[0mdeltachat-ffi/src/lib.rs:2405:5[0m
[0m     [0m[0m[1m[38;5;12m|[0m
[0m[1m[38;5;12m2405[0m[0m [0m[0m[1m[38;5;12m|[0m[0m [0m[0m    Box::from_raw(a);[0m
[0m     [0m[0m[1m[38;5;12m| [0m[0m    [0m[0m[1m[38;5;9m^^^^^^^^^^^^^^^^^[0m
[0m     [0m[0m[1m[38;5;12m|[0m
[0m     [0m[0m[1m[38;5;12m= [0m[0m[1mnote[0m[0m: call `drop(from_raw(ptr))` if you intend to drop the `Box`[0m

[0m[1m[38;5;9merror[0m[0m[1m: unused return value of `Box::<T>::from_raw` that must be used[0m
[0m    [0m[0m[1m[38;5;12m--> [0m[0mdeltachat-ffi/src/lib.rs:2586:5[0m
[0m     [0m[0m[1m[38;5;12m|[0m
[0m[1m[38;5;12m2586[0m[0m [0m[0m[1m[38;5;12m|[0m[0m [0m[0m    Box::from_raw(chatlist);[0m
[0m     [0m[0m[1m[38;5;12m| [0m[0m    [0m[0m[1m[38;5;9m^^^^^^^^^^^^^^^^^^^^^^^^[0m
[0m     [0m[0m[1m[38;5;12m|[0m
[0m     [0m[0m[1m[38;5;12m= [0m[0m[1mnote[0m[0m: call `drop(from_raw(ptr))` if you intend to drop the `Box`[0m

[0m[1m[38;5;9merror[0m[0m[1m: unused return value of `Box::<T>::from_raw` that must be used[0m
[0m    [0m[0m[1m[38;5;12m--> [0m[0mdeltachat-ffi/src/lib.rs:2731:5[0m
[0m     [0m[0m[1m[38;5;12m|[0m
[0m[1m[38;5;12m2731[0m[0m [0m[0m[1m[38;5;12m|[0m[0m [0m[0m    Box::from_raw(chat);[0m
[0m     [0m[0m[1m[38;5;12m| [0m[0m    [0m[0m[1m[38;5;9m^^^^^^^^^^^^^^^^^^^^[0m
[0m     [0m[0m[1m[38;5;12m|[0m
[0m     [0m[0m[1m[38;5;12m= [0m[0m[1mnote[0m[0m: call `drop(from_raw(ptr))` if you intend to drop the `Box`[0m

[0m[1m[38;5;9merror[0m[0m[1m: unused return value of `Box::<T>::from_raw` that must be used[0m
[0m    [0m[0m[1m[38;5;12m--> [0m[0mdeltachat-ffi/src/lib.rs:2991:5[0m
[0m     [0m[0m[1m[38;5;12m|[0m
[0m[1m[38;5;12m2991[0m[0m [0m[0m[1m[38;5;12m|[0m[0m [0m[0m    Box::from_raw(msg);[0m
[0m     [0m[0m[1m[38;5;12m| [0m[0m    [0m[0m[1m[38;5;9m^^^^^^^^^^^^^^^^^^^[0m
[0m     [0m[0m[1m[38;5;12m|[0m
[0m     [0m[0m[1m[38;5;12m= [0m[0m[1mnote[0m[0m: call `drop(from_raw(ptr))` if you intend to drop the `Box`[0m

[0m[1m[38;5;9merror[0m[0m[1m: unused return value of `Box::<T>::from_raw` that must be used[0m
[0m    [0m[0m[1m[38;5;12m--> [0m[0mdeltachat-ffi/src/lib.rs:3712:5[0m
[0m     [0m[0m[1m[38;5;12m|[0m
[0m[1m[38;5;12m3712[0m[0m [0m[0m[1m[38;5;12m|[0m[0m [0m[0m    Box::from_raw(contact);[0m
[0m     [0m[0m[1m[38;5;12m| [0m[0m    [0m[0m[1m[38;5;9m^^^^^^^^^^^^^^^^^^^^^^^[0m
[0m     [0m[0m[1m[38;5;12m|[0m
[0m     [0m[0m[1m[38;5;12m= [0m[0m[1mnote[0m[0m: call `drop(from_raw(ptr))` if you intend to drop the `Box`[0m

[0m[1m[38;5;9merror[0m[0m[1m: unused return value of `Box::<T>::from_raw` that must be used[0m
[0m    [0m[0m[1m[38;5;12m--> [0m[0mdeltachat-ffi/src/lib.rs:3866:5[0m
[0m     [0m[0m[1m[38;5;12m|[0m
[0m[1m[38;5;12m3866[0m[0m [0m[0m[1m[38;5;12m|[0m[0m [0m[0m    Box::from_raw(lot);[0m
[0m     [0m[0m[1m[38;5;12m| [0m[0m    [0m[0m[1m[38;5;9m^^^^^^^^^^^^^^^^^^^[0m
[0m     [0m[0m[1m[38;5;12m|[0m
[0m     [0m[0m[1m[38;5;12m= [0m[0m[1mnote[0m[0m: call `drop(from_raw(ptr))` if you intend to drop the `Box`[0m

[0m[0m[1m[31merror[0m[1m:[0m could not compile `deltachat_ffi` due to 9 previous errors
make[2]: *** [CMakeFiles/lib_deltachat.dir/build.make:74: target/release/libdeltachat.so] Error 101
make[1]: *** [CMakeFiles/Makefile2:83: CMakeFiles/lib_deltachat.dir/all] Error 2
make: *** [Makefile:136: all] Error 2
</details>

So it is broken for my system too.

## Some details about the system

```bash
nix \
run \
nixpkgs#nix-info -- --markdown
```
 - system: `"x86_64-linux"`
 - host os: `Linux 5.15.53, NixOS, 22.11 (Raccoon), 22.11.20220715.65fae65`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.10.2`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`


```bash
nix flake metadata nixpkgs
```

It outputs:
```bash
Resolved URL:  github:NixOS/nixpkgs/nixpkgs-unstable
Locked URL:    github:NixOS/nixpkgs/03a00f66fc4e893dccba1579df6d0c83852e1c2c
Description:   A collection of packages for the Nix package manager
Path:          /nix/store/a82y4w3v7qin8rsvh1rd50idaq0vspfa-source
Revision:      03a00f66fc4e893dccba1579df6d0c83852e1c2c
Last modified: 2022-10-25 06:04:09
Inputs:
```

--- a/pkgs/applications/networking/instant-messengers/deltachat-desktop/default.nix
+++ b/pkgs/applications/networking/instant-messengers/deltachat-desktop/default.nix
@@ -4,6 +4,7 @@
 , buildGoModule
 , esbuild
 , fetchFromGitHub
+, fetchpatch
 , libdeltachat
 , makeDesktopItem
 , makeWrapper
@@ -31,6 +32,13 @@ let
       name = "${old.pname}-${version}";
       hash = "sha256-4rpoDQ3o0WdWg/TmazTI+J0hL/MxwHcNMXWMq7GE7Tk=";
     };
+    patches = [
+      (fetchpatch {
+        name = "turn-off-hard-errors-for-lints.patch";
+        url = "https://github.com/deltachat/deltachat-core-rust/commit/7598c50dbaa2abcbd417d96a02743269f666597b.patch";
+        hash = "sha256-Xss44v6Wf6mL3FK9hH+oFYZ0fBA9rSh4wDrr7nSUibQ=";
+      })
+    ];
   });
   esbuild' = esbuild.override {
     buildGoModule = args: buildGoModule (args // rec {

